Tests/update_content_data.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+import os
+import ast
+import demisto_client
+
+from Tests.test_utils import print_error
+
+
+def options_handler():
+    parser = argparse.ArgumentParser(description='Utility to upload new content')
+    parser.add_argument('-u', '--user', help='The username for the login', required=True)
+    parser.add_argument('-p', '--password', help='The password for the login', required=True)
+    parser.add_argument('-s', '--server', help='The server to connect to (leaving out the protocol e.g.'
+                        ' without \'https://\')', required=True)
+    parser.add_argument('-up', '--content_zip', help='Path to new content zipfile to upload', required=True)
+
+    options = parser.parse_args()
+
+    return options
+
+
+def upload_content(server, username, password, content_zip_path):
+    try:
+        # Configure Demisto Client and make request to upload content zip file
+        c = demisto_client.configure(base_url=server, username=username, password=password, verify_ssl=False)
+        file_path = os.path.abspath(content_zip_path)
+        files = {'file': file_path}
+        header_params = {'Content-Type': 'multipart/form-data'}
+
+        msg = '\nMaking "POST" request to server - "{}" to upload'.format(server)
+        msg += ' the content zip file "{}"'.format(content_zip_path)
+        print(msg)
+        response_data, status_code, _ = c.api_client.call_api(resource_path='/content/upload', method='POST',
+                                                              header_params=header_params, files=files)
+
+        if status_code >= 300 or status_code < 200:
+            result_object = ast.literal_eval(response_data)
```

Comment:
why do we need ast.literal_eval here? what's the benefit of it's use

Reply:
I saw that this was how `demisto-py` handled requests. Since I'm making a request using the `demisto-py` client, I thought it made sense to use their error handling.